linux/keyman-config/debian/man/get_kmp.1

```diff
@@ -0,0 +1,24 @@
+.\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.47.6.
```

Comment:
I removed the generated man pages.

Reply:
I would prefer removing all generated files from the repo. Make sure the dependencies are clear and it won't be a problem.

Reply:
All autotool etc generated files removed in 12ab8f9 
I haven't messed with the lex and yacc generated files in kmflcomp since it will be going away soon, plus for these it can be useful for older systems to build the files generated by a newer flex and bison than they have